### ISSUE TITLE
bug: ion-menu-button does not appear in ios

### ISSUE THREAD

**Issue by carlosnewmusic** [ionic-team/ionic#18422]
**Ionic version:**
ionic v4.4.2~4.6

**Current behavior:**
![ion-menu-button ios fail](https://user-images.githubusercontent.com/19379457/58640451-62595500-82f9-11e9-8227-d9169b544e05.jpg)
![ion-menu-button ios fail2](https://user-images.githubusercontent.com/19379457/58640462-6a18f980-82f9-11e9-8049-2f5080be2419.jpg)

**Expected behavior:**

![image](https://user-images.githubusercontent.com/19379457/58640925-69cd2e00-82fa-11e9-9f64-4142418bcf2a.png)

** Steps to play: **
Running on ios browsers

** Related code: **
```<ion header>
<ion-toolbar>
<ion button slot = "start">
<ion-menu-button> </ ion-menu-button>
</ ion-buttons>
<ion-title> Login </ ion-title>
</ ion-toolbar>
</ ion-header>```

** Ionic information: **
Ionic:

ionic (ionic CLI): 4.12.0 (C: \ blablabla \ npm \ node_modules \ ionic)
Ionic frame: @ ionic / angular 4.4.2
@ angular-devkit / build-angular: 0.13.9
@ angular-devkit / schematics: 7.3.9
@ angular / cli: 7.3.9
@ Ionic / angular tool kit: 1.5.1

System:

NodeJS: v10.15.3 (C: \ Program Files \ nodejs \ node.exe)
npm: 6.4.1
OS: Windows 10

**Comment by liamdebeasi** [comment]
Hi there,

Thanks for the issue! Can you provide a repository with the code required to reproduce this issue?

Thanks!

**Comment by carlosnewmusic** [comment]
`<ion-header >
    <ion-toolbar>
        <ion-buttons slot="start">
            <ion-menu-button></ion-menu-button>
          </ion-buttons>
      <ion-title>Fichaje</ion-title>
   </ion-toolbar>
</ion-header>
`
this also fails in version 4.5 of ionic
the temporary solution that I have applied is
`<ion-header >
    <ion-toolbar **mode="md"**>
        <ion-buttons slot="start">
            <ion-menu-button **mode="md"**></ion-menu-button>
          </ion-buttons>
      <ion-title>Fichaje</ion-title>
    </ion-toolbar>
  </ion-header>`

**Comment by liamdebeasi** [comment]
Hi there,

Thanks for the follow up. Can you provide a full code reproduction? I am not able to reproduce the issue with that code snippet.

Thanks!

**Comment by carlosnewmusic** [comment]
you paste this into any created page and the error will be reproducedhttps://codepen.io/anon/pen/mZPjRG

**Comment by liamdebeasi** [comment]
Hi there,

Thanks for the follow up. I used that code in the Ionic side-menu starter app, but was unable to reproduce the issue you are seeing. Like I mentioned before, a full code reproduction would be helpful here. 🙂 

Thanks!

**Comment by liamdebeasi** [comment]
Additionally, are you seeing this issue when running on an iOS device, or is it only happening when using Ionic Lab?

**Comment by carlosnewmusic** [comment]
> Additionally, are you seeing this issue when running on an iOS device, or is it only happening when using Ionic Lab?

It happens in both IOS and Ionic Lab, and I can not provide all the code as it is a proprietary application

**Comment by liamdebeasi** [comment]
Thanks for the follow up! I don't need the proprietary part of your codebase, I just need a minimal amount of Ionic/side menu code so I can see the bug happening 🙂. If you could create a repo on GitHub with that, that would be very helpful.

Thanks!

**Comment by carlosnewmusic** [comment]
with that on top of any ion-content and running it on browsers you can reproduce it https://pastebin.com/pMd51BQq

**Comment by liamdebeasi** [comment]
Hi there,

Like I mentioned before, the code snippet you have provided was not enough for me to be able to reproduce this issue. If you could provide a repository with only the code required to reproduce this issue, that would be great (I do not need your proprietary code).

Thanks!

**Comment by corysmc** [comment]
Hey @liamdebeasi here's a similar issue - perhaps related.
When using the split-pane the ion-menu doesn't show up when the split pane menu is open (which it shouldn't...) however, there's still space there where the icon used to be

v4.6.0
https://codepen.io/corysmc/pen/XwExJJ?editors=1010
![image](https://user-images.githubusercontent.com/6452188/60284207-34e7d300-98c8-11e9-98c5-cf7eb37070ee.png)



This worked fined on 4.1.0
https://codepen.io/corysmc/pen/bPYvYJ?editors=1010
![image](https://user-images.githubusercontent.com/6452188/60284147-14b81400-98c8-11e9-8750-cb4abee472d6.png)

**Comment by brandyscarney** [comment]
Thanks @corysmc! There's a PR to fix that bug here: https://github.com/ionic-team/ionic/pull/18564

The original issue here may be that the menu button is taking on the same color as the toolbar so it is visible, but since they are the same color you can't see it. I'd be interested in seeing how you are setting that color on the toolbar @carlosnewmusic. If you are using global theme variables there may be a problem there. For example, if you're setting the following:

```
--ion-toolbar-background: var(--ion-color-primary);
```

You would not be able to see the menu button since in iOS it uses the primary color by default. This is all speculation though as we cannot reproduce this.

I've taken your code from the comments and placed it into a working Codepen and I am able to see the menu button: https://codepen.io/brandyscarney/pen/dBJeLL

If you could update this Codepen to include any CSS variables that you are overriding or provide us with additional code to reproduce we can help, but unfortunately we cannot fix an issue that we are not able to recreate. Thank you!

**Comment by simonhaenisch** [comment]
@corysmc I think it's a regression from 4.4 to 4.5, introduced by #18434 (see https://github.com/ionic-team/ionic/issues/18666#issuecomment-506987583).

**Comment by carlosnewmusic** [comment]
> Thanks @corysmc! There's a PR to fix that bug here: #18564
> 
> The original issue here may be that the menu button is taking on the same color as the toolbar so it is visible, but since they are the same color you can't see it. I'd be interested in seeing how you are setting that color on the toolbar @carlosnewmusic. If you are using global theme variables there may be a problem there. For example, if you're setting the following:
> 
> ```
> --ion-toolbar-background: var(--ion-color-primary);
> ```
> 
> You would not be able to see the menu button since in iOS it uses the primary color by default. This is all speculation though as we cannot reproduce this.
> 
> I've taken your code from the comments and placed it into a working Codepen and I am able to see the menu button: https://codepen.io/brandyscarney/pen/dBJeLL
> 
> If you could update this Codepen to include any CSS variables that you are overriding or provide us with additional code to reproduce we can help, but unfortunately we cannot fix an issue that we are not able to recreate. Thank you!
I went back to check the problem and in version 4.5 does not hide the menu icon, but if you focus the titles thing that on small mobiles if you have something right on the ion-toolbar overlays it
https://stackblitz.com/edit/ionic-jbzes5?file=src%2Fapp%2Fpages%2Ffichaje%2Ffichaje.page.html

**Comment by liamdebeasi** [comment]
Hi everyone,

Is this issue still reproducible in Ionic Framework 4.11.1?

**Comment by simonhaenisch** [comment]
I'm not sure but I think this has also been solved by #18702 along with the issue in #18666?

**Comment by liamdebeasi** [comment]
Yeah that could be it. @carlosnewmusic are you still running into this issue on the latest version of Ionic?
